tests/Checker/AJAX_Runner_Tests.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Tests for the Checks class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\AJAX_Runner;
+use WordPress\Plugin_Check\Checker\Check_Result;
+
+class AJAX_Runner_Tests extends WP_UnitTestCase {
```

Comment:
Just an observation, but since all of these are filtering the `wp_doing_ajax` value, is there any value in moving this shared code to a fixture that runs before each test? I also wonder if there's any value in having a test case that confirms the expected behavior if `wp_doing_ajax` is `false`?

Reply:
Thanks @joemcgill I've added an additional test to check behaviour when `wp_doing_ajax` is `false`. We can potentially refactor the tests here in future to reduce duplicate code here.